File: LeetCode/3.无重复字符的最长子串/Solution.py
class Solution:
    def lengthOfLongestSubstring(self, s: str) -> int:
        max = 0
        current = 0
        l = len(s)
        index = 0
        for i in range(0,l):
            current = 1
            for j in range(index,i):
                if (s[i] == s[j]):
                    index = j + 1
                    break
                current +=1
            if max < current:
                max = current
        return max

File: LeetCode/3.无重复字符的最长子串/Solution.kt
class Solution {
    fun lengthOfLongestSubstring(s: String): Int {
        var len = s.length
        var current = 0
        var max = 0
        var index = 0
        for (i in 0..len - 1){
            current = 1
            for (j in index..i - 1){
                if (s[i] == s[j]){
                     index = j + 1
                     break
                }
                current++
            }
            if (max < current){
                max = current
            }
        }
        return max
    }
}